app/views/stats/range.html.erb

```diff
@@ -1,7 +1,53 @@
 <h1>Activity</h1>
 
-<p>From <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %></p>
-
+<h5>From <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %></h5>
+<h4><b> Select a range below to view stats within the range </b></h4>
+<%= form_tag request.url, method: 'get' do %>
+    <div class="row">
+    <div class="col-md-8" style="background:#F5F6F8;">
```

Comment:
Here you could use the `well` css class from Bootstrap instead of a background css style. Take a look! 

https://getbootstrap.com/docs/3.3/